packages/storybook-blocks/package.json

```diff
@@ -40,7 +40,7 @@
     "prettier:fix": "prettier --write \"{,!(node_modules)/**/}*.{js,jsx,ts,tsx}\"",
     "test": "jest --passWithNoTests",
     "test:update": "yarn test -- -u",
-    "test:stories": "testing=storybook TEST_END_FILES=jest-story jest --passWithNoTests",
+    "test:stories": "testing=storybook TEST_END_FILES=test-story jest --passWithNoTests",
```

Comment:
I think it's redundant, we're not testing it, and also storyshots addon is deprecated
and anyway what the test did is:

```
**renderOnly**
Just render the story, don't check the output at all. This is useful as a low-effort way of smoke testing your components to ensure they do not error.
```

we already have this with Chromatic

Reply:
So you can remove `addon-storyshots` package as well + `stories.jest-story.ts` file